app/services/person_populator.rb

```diff
@@ -0,0 +1,47 @@
+class PersonPopulator
```

Comment:
Let's add a TODO here,
to specify this will be removed after the profile/person refator

Reply:
Done